### DISCUSSION TITLE
Stop autocorrecting (c) to copyright symbol

### DISCUSSION THREAD

**Discussion by jsjoshua** [nocodb/nocodb#8502] [Q&A]
When typing or pasting bulleted text into a long-text field, NocoDB automatically changes "(c)" into the copyright symbol.  Is there any way to correct or change this behavior?

**Comment by dstala** [comment]
@jsjoshua I could see this behaviour. Will discuss with the team if there is any escape sequence that could help in this case. adding a space helps ( c ), but that's not what you are looking for I believe.

**Reply by jsjoshua** [reply]
I'm adding the space manually, but I'm pasting large blocks of text in outline format, so the (c) comes up a lot.  Thanks.
